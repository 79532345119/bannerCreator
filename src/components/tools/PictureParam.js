import React from "react"


export const PictureParam =(props) => {


    return (
        <div className = "tools_elem">
        <label htmlFor="picLink">ссылка на картинку</label>
        <input type="text" placeholder="ссылка на картинку" 
               id="picLink" value={props.picLink} onChange={e=>props.setPicLink(e.target.value)}/>                 
        <label htmlFor="picScale">масштабировать картинку</label>
        <span>
            <input className="sm" type="number" id="picScale" value={props.picScaleX}
                onChange={e=>props.setPicScaleX(e.target.value)} />
            <input className="sm" type="number" value={props.picScaleY}
                onChange={e=>props.setPicScaleY(e.target.value)} />
        </span>

        </div>
    )
}