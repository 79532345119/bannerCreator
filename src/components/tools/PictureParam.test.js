import React from "react"
import {create} from "react-test-renderer"
import {PictureParam} from "./PictureParam"


describe("PictureParam component", ()=> {
    test("PictureParam component renders correctly", ()=> {
        const component = create(<PictureParam/>);
        expect(component).toMatchSnapshot();
    })
})