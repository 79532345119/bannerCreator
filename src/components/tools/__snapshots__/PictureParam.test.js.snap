// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PictureParam component PictureParam component renders correctly 1`] = `
<div
  className="tools_elem"
>
  <label
    htmlFor="picLink"
  >
    ссылка на картинку
  </label>
  <input
    id="picLink"
    onChange={[Function]}
    placeholder="ссылка на картинку"
    type="text"
  />
  <label
    htmlFor="picScale"
  >
    масштабировать картинку
  </label>
  <span>
    <input
      className="sm"
      id="picScale"
      onChange={[Function]}
      type="number"
    />
    <input
      className="sm"
      onChange={[Function]}
      type="number"
    />
  </span>
</div>
`;
